tests/providers/databricks/operators/test_databricks.py

```diff
@@ -1397,6 +1397,70 @@ def test_exec_failure_if_job_id_not_found(self, db_mock_class):
 
         db_mock.find_job_id_by_name.assert_called_once_with(JOB_NAME)
 
+    @mock.patch("airflow.providers.databricks.operators.databricks.DatabricksHook")
+    def test_cancel_previous_runs(self, db_mock_class):
+        run = {"notebook_params": NOTEBOOK_PARAMS, "notebook_task": NOTEBOOK_TASK, "jar_params": JAR_PARAMS}
+        op = DatabricksRunNowOperator(task_id=TASK_ID, job_id=JOB_ID, cancel_previous_runs=True, json=run)
+        db_mock = db_mock_class.return_value
+        db_mock.run_now.return_value = 1
```

Comment:
I feel like we probably want to change it to `RUN_ID`. It would make it easier to read. If possible, all other places in this test files as well.

```suggestion
        db_mock.run_now.return_value = RUN_ID
```

Reply:
This is done.